docs/using/recipes.md

```diff
@@ -530,10 +530,11 @@ USER root
 # autosklearn requires swig, which no other image has
 RUN apt-get update && \
     apt-get install -y --no-install-recommends swig && \
+    apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
+
 USER $NB_UID
-ARG py_ver=3.6
 
-RUN pip install auto-sklearn
+RUN pip install --quiet auto-sklearn
```

Comment:
Sorry but I have to mention a remark that has been done just a couple of days before on another case #1082. It's a good practice to add the `--no-cache-dir` option to reduce the image size. Maybe you could add it?

Reply:
No problem, added!

Reply:
Tks ping me when it's done.

Reply:
Done, thanks